src/endpoints/sources.js

```diff
@@ -292,6 +310,26 @@ function pathParts(path = "") {
 }
 
 
+/**
+ * Generate an Express endpoint that sends a group overview
+ * determined by the request.
+ *
+ * @returns {expressEndpointAsync}
+ */
+function sendGroupOverview() {
+  return async (req, res) => {
```

Comment:
Now that this doesn't take any parameters, there's no need for the indirection of returning a function. The `sendGroupOverview` symbol can have the `async (req, res)` signature directly instead. Ditto with `receiveGroupOverview`.

Reply:
Done: a28c3c97475cac2c209e4d02cf8cab678dc6717a, 6f3867d88476b6d7c191faf0b2f54d24d98d16a9